content/sensu-go/5.16/release-notes.md

```diff
@@ -48,6 +49,41 @@ Read the [upgrade guide][1] for information about upgrading to the latest versio
 
 ---
 
+## 5.16.0 release notes
+
+**December 11, 2019** &mdash; The latest release of Sensu Go, version 5.16.0, is now available for download.
```

Comment:
```suggestion
**December 12, 2019** &mdash; The latest release of Sensu Go, version 5.16.0, is now available for download.
```

Reply:
I'll hold off on this until the release date is firmed up.

Reply:
Good point 👍